web/src/components/modal-event-details/index.tsx

```diff
@@ -9,6 +9,7 @@ import { RepeatEvent } from '@constants';
 import { yupResolver } from '@hookform/resolvers/yup';
 import { IModalEventDetails } from '@interfaces';
 import { TextField, Tooltip } from '@mui/material';
+import * as MuiAutocompleteSelect from '@mui/material/Autocomplete';
```

Comment:
Can't we just import it like this? 

`import { TextField, Tooltip, MuiAutocompleteSelect } from '@mui/material';`